File: 351001/Budnikov/publisher/src/models/editor.py
from tortoise import fields, models


class Editor(models.Model):
    id = fields.IntField(pk=True)
    login = fields.CharField(max_length=64, unique=True)
    password = fields.CharField(max_length=128)
    firstname = fields.CharField(max_length=64)
    lastname = fields.CharField(max_length=64)

    issues: fields.ReverseRelation["Issue"]

    class Meta:
        table = "tbl_editor"

File: 351004/Purenok/Task310/src/main/java/com/example/Task310/bean/Editor.java
package com.example.Task310.bean;

import jakarta.persistence.*;
import lombok.*;
import java.util.List;

@Entity
@Table(name = "tbl_editor", schema = "distcomp")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Editor {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, unique = true, length = 64)
    private String login;

    @Column(nullable = false, length = 128)
    private String password;

    @Column(nullable = false, length = 64)
    private String firstname; // Обязательно для тестов

    @Column(nullable = false, length = 64)
    private String lastname;  // Обязательно для тестов

    @OneToMany(mappedBy = "editor", cascade = CascadeType.ALL)
    private List<Story> stories;
}